567 contained examples of work

ChristophEwertowski opened the pull request:
Resolves https://github.com/hbz/lobid-resources/issues/567

Also https://github.com/hbz/lobid-resources/issues/516 and https://github.com/lobid/lodmill/issues/784

ChristophEwertowski commented:
Will put the changes later on test. For `exampleOfWork` of MAB `304-1.a` see [CT003012479 (commit change)](https://github.com/hbz/lobid-resources/pull/671/files#diff-ca3d71107932d412300a2ad41d2b812c). For `303-1` see [HT018781534 (commit change)](https://github.com/hbz/lobid-resources/pull/671/files#diff-9dcab9b0d92cdc0904d7aa5a68497b6d).
For `containsExampleOfWork` (MAB `303t1`) see [HT018857620 (commit change)](https://github.com/hbz/lobid-resources/pull/671/files#diff-5dd561891677f7164f7d7c47a1d0438b).

Is it necessary to include the type `Work` if the properties are called "exampleOf*Work*" and "containsExampleOf*Work*"? Because there is no GND-Id for the creators I also added the living statistics at the end of the names. I didn't include MAB `521` because it has nothing to do with work titles.

acka47 commented on src/test/resources/jsonld/HT018857620:
```diff
@@ -0,0 +1,168 @@
+    "type" : [ "Work" ]
+  }, {
+    "creatorOfWork" : "Wagner, Richard 1813-1883",
+    "id" : "http://d-nb.info/gnd/300169914",
```
This looks very similar to the one above but it has a different GND ID. To make the difference between those clear, we will also have to add the sub title from sub field `u`, here  `<subfield code="u">Trauermarsch</subfield>`. See also https://github.com/hbz/lobid-resources/issues/556#issue-253656189.

acka47 commented on src/test/resources/jsonld/HT018781534:
```diff
@@ -30,6 +30,10 @@
   "edition" : [ "Score and parts" ],
+  "exampleOfWork" : {
+    "id" : "http://d-nb.info/gnd/107360974X",
+    "label" : "Momento capriccioso"
```
It would be nice to also add the information from subfields `p` and `d` like we do e.g. at https://github.com/hbz/lobid-resources/pull/671/files#diff-5dd561891677f7164f7d7c47a1d0438b.

acka47 commented:
I commented on the examples.

> Is it necessary to include the type `Work` if the properties are called "exampleOf*Work*" and "containsExampleOf*Work*"?

We should have a type for every object if possible. So if it doesn't create any problems I suggest to keep/add it.

> Because there is no GND-Id for the creators I also added the living statistics at the end of the names. 

Very good. As noted in the comment on https://github.com/hbz/lobid-resources/pull/671/files#diff-9dcab9b0d92cdc0904d7aa5a68497b6d it is still missing for some.

ChristophEwertowski commented:
Added the improvements to test. See [HT018857620 (test, json)](http://test.lobid.org/resources/HT018857620.json) for subfield `u`. There are also cases where the subfield appears multiple times.
See [HT018781534 (test, json)](http://test.lobid.org/resources/HT018781534.json) for additional `creatorOfWork` and type `Work`.
Also added MAB 521 where no 303t exists (example [HT018138676 (test, json)](http://test.lobid.org/resources/HT018138676.json)).
CT003012479 misses exampleOfWork ([json](http://test.lobid.org/resources/CT003012479.json)) although it's in the branch. Have to ask @dr0i about it tomorrow.

acka47 commented:
+1 from except for one last small change. I checked the GND and it uses periods as delimiter between title and subtitle(s), see e.g. http://d-nb.info/gnd/300169914. I think we should also do so.

Also, http://test.lobid.org/resources/CT003012479.json looks good for me (except for the encoding which generally is problematic on test):

```json
{
  "id" : "http://lobid.org/resources/CT003012479#!",
  "exampleOfWork" : {
    "label" : "<<La>> Belisa Ossia La FedeltÃ¡ Riconosciuta",
    "type" : [ "Work" ]
  }
}
```

ChristophEwertowski commented:
For the new changes look at [HT018857620 (json, test)](http://test.lobid.org/resources/HT018857620.json). For example:
* `"label" : "Sinfonien, D 759 (h-Moll)"` ([GND](http://d-nb.info/gnd/300141777))
* `"label" : "La damnation de Faust. Invocation à la nature, Singstimme, Orchester",` ([GND](http://d-nb.info/gnd/1081431059))

dr0i commented:
+1

dr0i commented:
Deployed to staging, see e.g. http://stage.lobid.org/resources/HT018857620.json.

dr0i commented:
Deployed to production, see e.g. http://lobid.org/resources/HT018857620.json.
Closed.
